CI/GitLab/.gitlab-ci.yml

```
@@ -0,0 +1,23 @@
+stages:
+  - build
+  - deploy
+
+cache:
+  paths:
+    - vendor/
+
+build:
+  stage: build
+  image: php:7.4
+  script:
+    - apt-get update
+    - apt-get install unzip
+    - curl -sS https://getcomposer.org/installer | php
+    - php composer.phar install --no-dev --no-progress --prefer-dist
+
+deploy:
+  stage: deploy
+  image: node:alpine
+  script:
+    - npm install -g serverless
+    - serverless deploy -v -s staging
```

```suggestion
    - serverless deploy -v
```

Let's remove the stage option in order to keep the example as simple (to the point) as possible. Deploying with a specific stage is not required.

---

Makes sense.